### DISCUSSION TITLE
Control Switch with PC

### DISCUSSION THREAD

**Discussion by ciberharry97** [cathery/sys-con#166] [Feature requests]
I would like to control the Switch with a controller connected to a PC. This would enable to play coop online with a friend that doesn't have the switch.

**Comment by amit27592** [comment]
If you know some development, you can use https://github.com/jakibaki/hid-mitm for this. Some code changes will be needed. (You can check sys-con implenmentation for the same.)

Check out input_pc.py in hid_mitm repo for how to actually transmit controller data over Lan.

**Comment by jabroany** [comment]
I concur! This function would be extremely wonderful.
